Packs/AzureCompute_v2/Integrations/AzureCompute_v2/AzureCompute_v2.yml

```diff
@@ -442,7 +446,7 @@ script:
     - contextPath: Azure.Subscription.State
       description: The state of the subscription.
       type: String
-  dockerimage: demisto/crypto:1.0.0.4187
+  dockerimage: demisto/crypto:1.0.0.7634
```

Comment:
@Itay4 since we no longer use the crypto package to get access token, can I replace the docker image for the Microsoft integrations to python3 image?